web/html/media/video-formats.md

```diff
@@ -48,23 +47,9 @@ AddType video/mp4 .mp4
 AddType video/webm .webm
 ```
 
-
 ---
 ## Practice
 
-Match the browser with the video file format that it works with:
-
-Mozilla Firefox & Google Chrome & Opera: ???
-Apple Safari & IE9: ???
-IE 6-8: ???
-
-* WebM & Ogv
-* MP4
-* No HTML5, Flash Only!
-
----
-## Revision
```

Comment:
Why did we remove this? This removes all questions completely. Did you intend on removing only the practice question?

Reply:
Removed the practice question because it is wrong. Replace it with the contents of the Revision question. Remove revision question